rfcs/text/0000-risk-input-fields.md

```diff
@@ -0,0 +1,119 @@
+# 0000: Risk Input fields
+
+- Stage: **0 (strawperson)**
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+### Summary
+This RFC aims to add a few general fields that, when defined on a document, will allow that document containing them to be consumed by Kibana's Risk Engine for the purposes of entity analytics.
+
+Broadly, we need two fields to enable this behavior:
+
+* `risk_score`, a "base" risk score (numeric, float) defined by the data producer that is used as the basis for calculating risk for the entity represented in the document
```

Comment:
@ajosh0504 is there any overlap between the existing `static_score` field and the concept defined here?

Reply:
So we've defined `static_score` as follows:

> A risk classification score obtained from outside the system, such as from some external Threat Intelligence Platform

Given the above definition, I think there could potentially be an overlap between the two, specifically if the data producer in the `risk_score` definition could be an external system. Might be good to clarify what data producer here entails.

Reply:
By "data producer" I mean "mechanism producing documents that intend to be scored by the risk engine." Did you intend for `static_score` to be used somehow in subsequent calculations, or is the intention more "display whatever is here as an auxiliary data point?" 

I was mainly asking the original intention to make sure that I wasn't misinterpreting/ignoring the `static_score` field. If it has a distinct use case from the one I'm proposing in this RFC with `risk_score`, I think we should keep them separate.

Reply:
I want to say it's the latter i.e. auxiliary data point, but @MikePaquette was the one who proposed the static_risk_score field when I was working on the Risk Fields RFC so tagging him for confirmation.